sites/public/src/components/listing/GetApplication.tsx

```diff
@@ -97,7 +100,7 @@ const GetApplication = (props: ApplicationsProps) => {
             <Button
               variant="primary"
               className="w-full mb-2"
-              href={props.onlineApplicationURL}
+              href={`${props.onlineApplicationURL}${props.preview ? "&preview=true" : ""}`}
```

Comment:
`onlineApplicationURL` is sometimes an external link - we should pass this in through the prop in `ListingView.tsx`

Reply:
Updated! Great catch 🐨